src/coreclr/nativeaot/docs/compiling.md

```diff
@@ -34,7 +33,7 @@ or by adding the following element to the project file:
 
 ## Cross-architecture compilation
 
-Native AOT toolchain allows targeting ARM64 on an x64 host and vice versa for both Windows and Linux. Cross-OS compilation, such as targeting Linux on a Windows host, is not supported. To target win-arm64 on a Windows x64 host, in addition to the `Microsoft.DotNet.ILCompiler` package reference, also add the `runtime.win-x64.Microsoft.DotNet.ILCompiler` package reference to get the x64-hosted compiler:
+Native AOT toolchain allows targeting ARM64 on an x64 host and vice versa for both Windows and Linux and is now supported in the SDK. Cross-OS compilation, such as targeting Linux on a Windows host, is not supported. To target win-arm64 on a Windows x64 host on an advanced scenario where the SDK support is not sufficient, in addition to the `Microsoft.DotNet.ILCompiler` package reference, also add the `runtime.win-x64.Microsoft.DotNet.ILCompiler` package reference to get the x64-hosted compiler:
```

Comment:
> Native AOT toolchain allows targeting ARM64 on an x64 host and vice versa for both Windows and Linux and is now supported in the SDK.

Do we have the cross-compilation via SDK (e.g. the stuff about sysroot) documented in the official docs?

The detail about using same version of explicit package reference for cross-compilation should be the very last in this topic. The topic should lead with using PublishAot property for cross-compilation now that it is supported by the SDK.

Reply:
Created an issue, https://github.com/dotnet/docs/issues/30889, for the official doc update